### PR TITLE
UNOMI-513: fix scoring plan recalculation

### PR DESCRIPTION
**Pull request by jkevan**
PR Title format: 

    [UNOMI-XXX] Pull request title with JIRA reference

**Please** add a meaningful description for your change here

----

**Please** following this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/UNOMI) filed 
       for the change (usually before you start working on it).  Trivial changes like typos do not 
       require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Format the pull request title like `[UNOMI-XXX] - Title of the pull request`
 - [ ] Provide integration tests for your changes, especially if you are changing the behavior of existing code or adding
       significant new parts of code.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why. 
       Copy the description to the related JIRA issue
 - [ ] Run `mvn clean install -P integration-tests` to make sure basic checks pass. A more thorough check will be 
        performed on your pull request automatically.
 
Trivial changes like typos do not require a JIRA issue (javadoc, project build changes, small doc changes, comments...). 
 
If this is your first contribution, you have to read the [Contribution Guidelines](https://unomi.apache.org/contribute.html)

If your pull request is about ~20 lines of code you don't need to sign an [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf) 
if you are unsure please ask on the developers list.

To make clear that you license your contribution under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)
you have to acknowledge this by using the following check-box.

 - [ ] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

### CODE REVIEW

File: services/src/main/java/org/apache/unomi/services/impl/segments/SegmentServiceImpl.java [hunk 1]
```diff
@@ -1271,6 +1260,25 @@ public void run() {
         schedulerService.getScheduleExecutorService().scheduleAtFixedRate(task, initialDelay,  TimeUnit.DAYS.toSeconds(1), TimeUnit.SECONDS);
     }
 
+    private void loadScripts() throws IOException {
+        Enumeration<URL> scriptsURL = bundleContext.getBundle().findEntries("META-INF/painless", "*.painless", true);
```
Comment (sergehuber) [inline services/src/main/java/org/apache/unomi/services/impl/segments/SegmentServiceImpl.java:1264]: Maybe put them in the /cxs subdirectory for consistency ?

**Review by sergehuber — APPROVED** [review]
Hi @jkevan thanks a lot for the fix. Storing the scripts is much better and having them in separate files is great ! I just found a minor issue with the directory for storing the scripts I hope you can change it? 
Best regards,
  Serge...

File: services/src/main/java/org/apache/unomi/services/impl/segments/SegmentServiceImpl.java (hunk 1 above)
Comment (jkevan) [reply services/src/main/java/org/apache/unomi/services/impl/segments/SegmentServiceImpl.java]: yes, thank you for the hint, I moved them to /cxs

Comment (sergehuber) [reply services/src/main/java/org/apache/unomi/services/impl/segments/SegmentServiceImpl.java]: Thanks you. Looks good to me now.

**Pull request merged**